package.json

```diff
@@ -143,6 +143,20 @@
           ],
           "default": "off"
         },
+        "sway-lsp.logging.level": {
+          "scope": "window",
+          "type": "string",
+          "description": "Set the log level for LSP server logs.",
+          "enum": [
+            "off",
+            "error",
+            "warn",
+            "info",
+            "debug",
+            "trace"
+          ],
+          "default": "info"
```

Comment:
hmm perhaps the default should be either `error` or `off`. I'm just worried that `info` might be a bit to granular for the average user as the default option.